scripts/perfect_hash.py

```diff
@@ -384,7 +391,12 @@ def __call__(self, data, quote=False):
         return "\n".join(l.rstrip() for l in aux.getvalue().split("\n"))
 
 
-def generate_code(keys, Hash=StrSaltHash, template=None, options=None):
+def generate_code(
+    keys: list[str],
+    Hash: type = StrSaltHash,
```

Comment:
sigh, github doesn't let me do suggestions here:
shoudl this be 

```
Hash: type[H] = StrSaltHash,
```
for consistency with `generate_hash()`

Reply:
Fixed in #534